pkgs/by-name/ga/garnet/package.nix

```diff
@@ -20,12 +20,7 @@ buildDotnetModule rec {
   projectFile = "main/GarnetServer/GarnetServer.csproj";
   nugetDeps = ./deps.nix;
 
-  dotnet-sdk =
-    with dotnetCorePackages;
-    combinePackages [
-      sdk_6_0
-      sdk_8_0
-    ];
+  dotnet-sdk = dotnetCorePackages.sdk_8_0;
```

Comment:
Already applied in https://github.com/NixOS/nixpkgs/commit/b810247f0aa89db4e75852c2074d98c40ef56c6b

Reply:
removed after rebase